source/dbo/Stored Procedures/Power BI Reporting/uspPopulateLookupTables.sql

```diff
@@ -31,6 +31,22 @@ BEGIN TRY
 	(3, 'NotConsistentWithTb', 'Not consistent with TB'),
 	(4, 'Awaiting', 'Awaiting')
 
+	TRUNCATE TABLE [dbo].[NtbsTransitionDateLookup]
+	INSERT INTO [dbo].[NtbsTransitionDateLookup] (PHEC, TransitionDate)
+	VALUES
+	('E45000001', NULL),
+	('E45000005', NULL),
+	('E45000009', '2021-07-19'),
```

Comment:
Should be 2021-07-09 for both Wales and the North East.  This means that a transfer request created on Saturday 10th (not that this will happen) will get picked up and included.

Reply:
updated!